spec/spec_helper.rb

```diff
@@ -21,6 +21,14 @@ def stub_batches
     end
   end
 
+  def stub_child_batches
```

Comment:
the official upstream recommendation is "don't try to unit test batches", and now I know why, there are a lot of internals to either set up in redis first, or navigate via mocking and stubbing; this isn't great, but it works, and it's in the same vein as the existing stubbing helpers that it is used with